lib/features/children/family_history/family_history.dart

```diff
@@ -0,0 +1,137 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app/features/children/family_history/family_history_logic/family_history_cubit.dart';
+import 'package:givt_app/features/children/family_history/models/allowance.dart';
+import 'package:givt_app/features/children/family_history/models/child_donation.dart';
+import 'package:givt_app/features/children/family_history/models/child_donation_helper.dart';
+import 'package:givt_app/features/children/family_history/models/history_item.dart';
+import 'package:givt_app/features/children/family_history/widgets/allowance_item_widget.dart';
+import 'package:givt_app/features/children/family_history/widgets/donation_item_widget.dart';
+import 'package:givt_app/features/children/overview/models/profile.dart';
+import 'package:givt_app/l10n/l10n.dart';
+import 'package:givt_app/utils/app_theme.dart';
+
+class FamilyHistory extends StatelessWidget {
+  const FamilyHistory({required this.children, super.key});
+  final List<Profile> children;
+  @override
+  Widget build(BuildContext context) {
+    final scrollController = ScrollController();
+    final historyCubit = context.read<FamilyHistoryCubit>();
+    final size = MediaQuery.of(context).size;
+    scrollController.addListener(() {
+      if (scrollController.position.maxScrollExtent ==
+          scrollController.position.pixels) {
+        if (historyCubit.state.status != HistroryStatus.loading) {
+          // Scrolled to end of list try to fetch more data
+          historyCubit.fetchHistory();
+        }
+      }
+    });
+    final locals = context.l10n;
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+         Padding(
+          padding:const EdgeInsets.symmetric(vertical: 8, horizontal: 25),
+          child: Text(
+            locals.childHistoryAllGivts,
+            style: const TextStyle(
```

Comment:
shall we use the following approach instead?:
`Theme.of(context).textTheme.blabla?.copyWith(...`